src/bridge/IrcHandler.ts

```diff
@@ -406,7 +406,15 @@ export class IrcHandler {
                 virtualMatrixUser.getId()
             ).invite(
                 room.getId(), invitee
-            );
+            ).catch(err => {
+                req.log.warn(
+                    `Failed to invite %s as the inviter user (reason: ${err}),
```

Comment:
You didn't provide an arg to %s, but we might as well use new template syntax anyway.
```suggestion
                    `Failed to invite ${invitee} as the inviter user (reason: ${err}),
```